openstack/swift/templates/etc/_nginx.conf.tpl

```diff
@@ -25,12 +25,17 @@ http {
 
     access_log  /var/log/nginx/access.log  main;
 
-    sendfile            on;
-    keepalive_timeout   65;
+{{ if .Values.client_timeout -}}
```

Comment:
`.Values` is not available in that context. Please use `$context`
Please use `{{ - if $cluster.client_timeout}}` otherwise `keepalive_timeout` lost is indentation